<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PianoTutorPanelBase</class>
 <widget class="QDockWidget" name="PianoTutor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>576</height>
   </rect>
  </property>
  <property name="floating">
   <bool>false</bool>
  </property>
  <property name="features">
   <set>QDockWidget::AllDockWidgetFeatures</set>
  </property>
  <property name="windowTitle">
   <string>PianoTutor</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="tutorEnabledCB">
        <property name="text">
         <string>Enable tutor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="tutorWaitCB">
        <property name="text">
         <string>Tutor waits for user</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="tutorLookAheadCB">
        <property name="text">
         <string>Tutor lookahead</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>LEDs stripe features</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>LEDs per key</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="lightsPerMeter">
             <property name="editable">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string>2.0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1.0</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>C-4 LED number</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="midCLight">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="baseSize">
              <size>
               <width>48</width>
               <height>0</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Usually half your stripe size minus 1, e.g., for a 144 stripe, try 71</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QCheckBox" name="backwardLayout">
             <property name="text">
              <string>LEDs laid out backwards</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tutorWizard">
             <property name="text">
              <string>Wizard...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="leftHandColor">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Left hand</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="rightHandColor">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Right hand</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="musescore.qrc"/>
 </resources>
 <connections/>
</ui>
